.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -1,59 +1,66 @@
-name: Bug report
-description: Let us know about functionality that is not working as expected
+name: 🐛 Bug Report
+description: Report a reproducible bug.
 title: "Bug report: <short description>"
-labels: ["bug: product"]
+labels: ["bug", "unconfirmed"]
 body:
   - type: markdown
     attributes:
       value: |
         Thank you for reporting the issue you are facing.
 
         Please complete the below form to ensure we have the necessary details to assist you.
-  - type: textarea
-    id: description
+  - type: checkboxes
+    id: prerequisites
     attributes:
-      label: Description
-      description: Provide a short description of the issue you are facing
-    validations:
-      required: true
+      label: Prerequisites
+      description: Please read and confirm before submitting an issue.
+      options:
+        - label: I am running the latest CosmPy version.
```

Comment:
Can you integrate the version here using `![cosmpy pypi version](https://img.shields.io/pypi/v/cosmpy)`? 

Something like:

```suggestion
        - label: I am running the latest CosmPy version ![cosmpy pypi version](https://img.shields.io/pypi/v/cosmpy).
```

Reply:
I would suggest making it a link to the PyPI page
```suggestion
        - label: I am running the latest CosmPy version [![cosmpy PyPI version](https://img.shields.io/pypi/v/cosmpy?label=cosmpy%20%28PyPI%29)].
```